src/main/scala/uk/gov/nationalarchives/consignmentexport/GraphQlApi.scala

```diff
@@ -35,6 +37,15 @@ class GraphQlApi(keycloak: KeycloakUtils,
     consignment = consignmentData.getConsignment
   } yield consignment
 
+  def updateConsignmentStatus(config: Configuration, consignmentId: UUID, statusType: String, status: String): IO[Option[Int]] = for {
+    token <- keycloak.serviceAccountToken(config.auth.clientId, config.auth.clientSecret).toIO
+    exportResult <- updateConsignmentClient.getResult(token, ucs.document, ucs.Variables(ConsignmentStatusInput(consignmentId, statusType, status)).some).toIO
+    consignmentStatus <-
+      IO.fromOption(exportResult.data)(new RuntimeException(s"No consignment found for the update consignment call for consignment $consignmentId ${exportResult.errorString}"))
```

Comment:
Maybe change the message of the `RuntimeException` to: `No data returned from the update consignment status call for consignment ...`  This would make it consistent with the other exception message